experimental/tinystr_neo/tests/serde.rs

```diff
@@ -0,0 +1,58 @@
+use tinystr_neo::*;
+
+// Tests largely adapted from `tinystr` crate
+// https://github.com/zbraniecki/tinystr/blob/4e4eab55dd6bded7f29a18b41452c506c461716c/tests/serde.rs
+
+macro_rules! test_roundtrip {
+    ($f:ident, $n:literal, $val:expr, $bytes:expr) => {
```

Comment:
Issue: You are not using `$bytes` anywhere

Reply:
Hmm, I thought I removed it, but perhaps i forgot to commit.

`bytes` used to be the actual bytes, but in this representation of `tinystr`, unlike the tinystr crate, the representation is always just flat bytes (yay) which is efficient and nice.